src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/VolleyManagementEntities.cs

```diff
@@ -4,10 +4,12 @@ namespace VolleyManagement.Data.MsSql.Context
     using System.Data.Entity.ModelConfiguration.Conventions;
     using Entities;
 
+#pragma warning disable S1200 // This class is a DB context - there could not be other way 
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)